0050    javascript-data-model/bookshelf.js

```diff
@@ -0,0 +1,21 @@
+var bookshelf = [
+  {
+    isbn: '978-1491929483',
+    title: 'Introduction to JavaScript Object Notation: A To-the-Point Guide to JSON',
+    author: 'Lindsay Bassett'
+  },
+  {
+    isbn: '978-1484218624',
+    title: 'JSON Quick Syntax Reference',
+    author: 'Wallace Jackson'
+  },
+  {
+    isbn: '978-0692232699',
+    title: 'Build APIs You Won\'t Hate: Everyone and their dog wants an API, so you should probably learn how to build them',
+    author: 'Phil Sturgeon & Laura Bohill'
+  }
+];
+
+console.log('Author of 1st book: ', bookshelf[1].author);
+console.log('ISBN of 2nd book: ', bookshelf[0].isbn);
+console.log('Title of 3rd book: ', bookshelf[2].title);
```

Comment:
these are not pulling the right indices. Let's double check them!

Reply:
I updated code and re-committed. 
![javascript-data-model-bookShelf](https://user-images.githubusercontent.com/126219504/228760948-61c2bb54-79e7-43a1-8311-730a15b6b083.png)